mkosi

```diff
@@ -4688,6 +4683,10 @@ def load_args(args: CommandLineArguments) -> CommandLineArguments:
     if args.qemu_headless and "console=ttyS0" not in args.kernel_command_line:
         args.kernel_command_line.append("console=ttyS0")
 
+    if args.bootable and args.distribution in (Distribution.opensuse, Distribution.mageia):
+        # TODO: Remove once dracut 045 is available in all these distros.
```

Comment:
dracut is already >045 in Tumbleweed ([also in Leap 15.2](https://software.opensuse.org/package/dracut), however, I'm only looking after TW mkosi variant). Perhaps this special case is unnecessary then.